Composer/packages/lib/shared/src/featureFlagUtils/Feature.tsx

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Do you have any plan for having a hook to do this? It's not always in a tsx that you need to get the value of a feature flag.

Reply:
I have added a useFeatureFlags() hook for those scenarios